uvod-do-programovani-2/slicing-metody-moduly/excs/zavody-2.md

```diff
@@ -0,0 +1,9 @@
+---
+title:  Závody podruhé
+demand: 5
+---
+
+Zadání je podobné jako u předchozího příkladu, ale nyní zkusíme výpočet provést pro všechny závodníky.
+
+* Nejprve vytvoř dvourozměrný seznam, kde na nulté pozici vnořeného seznamu je číslo běžce a na první pozici je čas běžce v minutách jako desetinné číslo.
```

Comment:
Myslí se zde explicitně přepsat seznam, anebo použít třeba list comprehension? Asi přepsat?

Reply:
Použít for cyklus a `append`